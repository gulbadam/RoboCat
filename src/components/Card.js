import React from 'react';
import "../styles/Card.css"
const Card= ({name, email, id}) => (
    <div className="tc bg-light-green grow dib f5-ns  black-95 pa3 ph2 ma2 ba b--light-silver shadow-3 br2 pointer">
        <img alt="" src={`https://robohash.org/${id}.png?set=set4`} />
    <div>
    <h2>{name}</h2>
    <p>{email}</p>
  
    </div>
    </div>)
 
export default Card;