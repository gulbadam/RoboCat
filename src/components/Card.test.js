import React from 'react';
import {shallow} from 'enzyme';
import Card from './Card';
it('expect to render Card Component', () => {
    expect(shallow(<Card />).length).toEqual(1)
});
it('expect to render Card snp', () => {
    expect(shallow( <Card /> )).toMatchSnapshot();
});