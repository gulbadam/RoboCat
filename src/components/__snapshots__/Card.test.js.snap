// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card snp 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Card />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <img
          alt=""
          src="https://robohash.org/undefined.png?set=set4"
/>,
        <div>
          <h2 />
          <p />
</div>,
      ],
      "className": "tc bg-light-green grow dib mw5 f5-ns  black-95 pa3 ph2 ma2 ba b--light-silver shadow-3 br2 pointer",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "",
          "src": "https://robohash.org/undefined.png?set=set4",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h2 />,
            <p />,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": undefined,
            },
            "ref": null,
            "rendered": null,
            "type": "h2",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": undefined,
            },
            "ref": null,
            "rendered": null,
            "type": "p",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <img
            alt=""
            src="https://robohash.org/undefined.png?set=set4"
/>,
          <div>
            <h2 />
            <p />
</div>,
        ],
        "className": "tc bg-light-green grow dib mw5 f5-ns  black-95 pa3 ph2 ma2 ba b--light-silver shadow-3 br2 pointer",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "",
            "src": "https://robohash.org/undefined.png?set=set4",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h2 />,
              <p />,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": undefined,
              },
              "ref": null,
              "rendered": null,
              "type": "h2",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": undefined,
              },
              "ref": null,
              "rendered": null,
              "type": "p",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
